src/component/background.js

```diff
@@ -77,6 +79,7 @@ SplitBackground.propTypes = {
 const wrapperStyles = StyleSheet.create({
   safe: {
     flex: 1,
+    paddingTop: Platform.OS === 'android' ? 25 : 0,
```

Comment:
Also added this to fix the statusbar on Android. See: https://stackoverflow.com/questions/51289587/react-native-how-to-use-safeareaview-for-android-notch-devices